src/capitalsFirst.test.js

```diff
@@ -2,62 +2,42 @@
 
 const capitalsFirst = require('./capitalsFirst');
 
-test('should work for "hey You, Sort me Already!"', () => {
+test('should work with common sentence', () => {
   expect(capitalsFirst('hey You, Sort me Already!'))
     .toBe('You, Sort Already! hey me');
 });
 
-test('should work for "You baby and Me"', () => {
-  expect(capitalsFirst('You baby and Me'))
+test('should work with common sentence 2', () => {
```

Comment:
what's the difference between "common sentence 2" and "common sentence 3"? Each task case should be testing different scenarios. Please rewrite descriptions to test different scenarios. If two test cases are testing the same scenario - remove one of them.

Example of different scenarios: 
1. all words are capitalized
1. none of the words are capitalized
1. one word capitalized. 
1. multiple words capitalized
1. none of the words start with the alphabetical letter.
etc.

Reply:
done